codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/ViewServiceSourceGeneratorSuite.scala

```diff
@@ -74,45 +73,43 @@ class ViewServiceSourceGeneratorSuite extends munit.FunSuite {
   }
 
   test("abstract source, transform_updates=true") {
-    val service = TestData.simpleViewService()
-    val packageName = "com.example.service"
+    val service = testData.simpleViewService()
 
-    val generatedSrc =
-      ViewServiceSourceGenerator.abstractView(service, packageName)
+    val generatedSrc = ViewServiceSourceGenerator.abstractView(service)
     assertNoDiff(
       generatedSrc,
       """|package com.example.service
          |
          |import com.akkaserverless.scalasdk.view.View
-         |import com.example.service.domain.EntityOuterClass
+         |import com.example.service.domain.EntityCreated
+         |import com.example.service.domain.EntityUpdated
          |
-         |abstract class AbstractMyServiceView extends View[ServiceOuterClass.ViewState] {
+         |abstract class AbstractMyServiceView extends View[ViewState] {
          |
          |
          |  def created(
-         |    state: ServiceOuterClass.ViewState, entityCreated: EntityOuterClass.EntityCreated): View.UpdateEffect[ServiceOuterClass.ViewState]
+         |    state: ViewState, entityCreated: EntityCreated): View.UpdateEffect[ViewState]
```

Comment:
this PR solves the imports for the things from `domain`, such as the `EntityCreated` here.